docs/examples/move/nft_marketplace/sources/clothing_store.move

```diff
@@ -0,0 +1,60 @@
+/// Module provides `mock` items for using them in marketplace and rental extensions.
+#[allow(lint(self_transfer))]
+module nft_marketplace::clothing_store {
+    use iota::package;
+    /// One Time Witness.
+    public struct CLOTHING_STORE has drop {}
+
+
+    fun init(otw: CLOTHING_STORE, ctx: &mut TxContext) {
```

Comment:
Do we need the OTW here? Looks like it can be removed.

Reply:
Do you mean we can replace it with

```
 fun init(ctx: &mut TxContext) {
        package::claim_and_keep(CLOTHING_STORE {}, ctx)
  }
```

Reply:
We have to have publisher anyway, publisher is required for TransferPolicy creation
So semantically and logically publisher should be created inside clothing_store as we use these items for marketplace stuff.